docs/get-started/create-and-debug-office-add-ins-in-visual-studio.md

```diff
@@ -16,7 +16,7 @@ To get started, use the  **Office Add-ins** project template in Visual Studio. T
 
 1. On the Visual Studio menu bar, choose  **File**,  **New**,  **Project**.
     
-2. In the list of project types under  **Visual C#** or **Visual Basic**, expand  **Office/SharePoint**, choose  **Office Add-ins**, and then choose  **Office Add-in**.
+2. In the list of project types under  **Visual C#** or **Visual Basic**, expand  **Office/SharePoint**, choose  **Office Add-ins**, and then choose  **Office Add-in**. If you do not see this option, it is likely the Office Developer Tools are not installed. You can download them [here](https://www.visualstudio.com/features/office-tools-vs.aspx).
```

Comment:
I suggest that we add this link up front rather than at step 2. Let's revise the sentence that begins with "To get started..." as follows:

"To get started, download the [Office Developer tools](https://www.visualstudio.com/features/office-tools-vs.aspx), if you haven't already, and use the... "